tests/routers/test_criteria.py

```
@@ -88,25 +89,25 @@ def test_delete_last_criteria_triggers_reversal(self, authenticated_client, db_s
             value="100",
             enabled=True
         )
         db_session.add(crit)
         db_session.commit()
         crit_id = crit.id
-        
+
         # Mock PathReverser
         from app.services.path_reverser import PathReverser
         reversed_called = []
         def mock_reverse(pid, db):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `mock_reverse`

(ANN202)